source/grid/grid_out.cc

```diff
@@ -2366,7 +2371,7 @@ GridOut::write_svg(const Triangulation<2, 2> &tria, std::ostream &out) const
                   if (svg_flags.label_level_number ||
                       svg_flags.label_cell_index)
                     out << ',';
-                  out << (int)cell->material_id();
+                  out << static_cast<int>(cell->material_id());
```

Comment:
you don't need the cast here any more

Reply:
Ah, I see. Can you make this clearer by casting to `signed int` then? I had imagined that we had that cast because originally, `material_id` was a `char` type.

Reply:
Yes, in the second last commit I use casts to `std::make_signed<types::subdomain_id>::type` instead.